crates/consensus/consensus/src/lib.rs

```diff
@@ -414,6 +416,15 @@ pub enum ConsensusError {
         child_gas_limit: u64,
     },
 
+    /// Error indicating that the block gas limit is above the allowed maximum.
+    ///
+    /// This error occurs when the gas limit is more than the specified maximum gas limit.
+    #[error("child gas limit {block_gas_limit} is above the maximum allowed limit ({MAXIMUM_GAS_LIMIT_BLOCK})")]
+    GasLimitInvalidBlockMaximum {
+        /// block gas limit.
+        block_gas_limit: u64,
+    },
```

Comment:
I think this comment is not completely correct right now-- the limit seems to apply to a transaction (looking for a reference), on the assumption that a transaction can use up all of the gas in the block. 

After 7825, this will no longer be the case though